tests/testImportCentroidAlgorithm.py

```diff
@@ -59,11 +60,11 @@ def testApplyCentroid(self):
         """Test that we can instantiate and play with SillyMeasureCentroid"""
 
         for imageFactory in (
-                             afwImage.MaskedImageF,
-                             ):
+            afwImage.MaskedImageF,
```

Comment:
I wonder why this is on a separate line. Were there multiple image types in this test originally?